src/inc/msquic.h

```diff
@@ -1293,6 +1293,11 @@ MsQuicOpenVersion(
 #if defined(__cplusplus) || defined(WIN32)
 
 _IRQL_requires_max_(PASSIVE_LEVEL)
+#ifdef WIN32
+__forceinline
+#else
+__attribute__((always_inline))
+#endif
```

Comment:
Let's just scope this to Posix for now:
```suggestion
#ifndef WIN32
__attribute__((always_inline))
#endif
```